engine/src/flutter/docs/contributing/Compiling-the-engine.md

```diff
@@ -117,11 +121,11 @@ For `--runtime-mode=profile` build, please also consider adding `--no-lto` optio
 
 These steps build the engine used by `flutter run` for iOS devices.
 
-Run the following steps, from the `src` directory created in the steps above:
+Run the following steps, from the `engine/src` directory:
 
-1. `git pull upstream main` in `src/flutter` to update the Flutter Engine repo.
+1. `git pull upstream master` to update the Flutter repo.
 
-2. `gclient sync` to update dependencies.
+2. `gclient sync -D` to update dependencies.
```

Comment:
I think this is run in the "root" folder and not "engine/src", but then again doesn't gclient find the root for you?